ionic-angular-template/src/auth/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, FormControl, Validators } from '@angular/forms';
+import { AuthService } from '../../services/auth.service';
+import { Router } from '@angular/router';
+import { AlertController } from '@ionic/angular';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+  selector: 'app-sign-up',
+  templateUrl: './sign-up.component.html',
+  styleUrls: ['./sign-up.component.scss'],
+})
+export class SignUpComponent implements OnInit {
+  private infoMessages: any;
+  public signUpForm: FormGroup;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private alertCtrl: AlertController,
+    private formBuilder: FormBuilder,
+    private tranlateService: TranslateService
+  ) { }
+
+  ngOnInit() {
+    this.tranlateService.get('Info-messages').subscribe((messages) => {
+      this.infoMessages = messages;
```

Comment:
I think that in that way, you are taking the whole translation file, which is not very good. Let's try to do it in this way:
```
this.translateService.stream([
      'wallet.misc.backup-phrase-copied',
    ]).subscribe(translations => {
      this.toastMessage = translations['wallet.misc.backup-phrase-copied'];
    });
```